docs/software/camera.md

```diff
@@ -62,45 +74,107 @@ The software can
 - observe based on motion detection
 - record a video with time lapse
 
-![MotionEye web interface screenshot](../assets/images/dietpi-software-camera-motioneye.png){: width="500" height="246" loading="lazy"}
-
-=== "Requirements"
-
-    The requirements are:
+from any RPi camera, USB camera or an IP camera network stream.
 
-    - Have any RPi-supported camera attached
-    - Have a Debian-supported USB camera attached
-    - Have access to an IP camera (access via network)
+![MotionEye web interface screenshot](../assets/images/dietpi-software-camera-motioneye.png){: width="500" height="246" loading="lazy"}
 
 === "Access the web interface"
 
-    The web interface is accessible via port 8765:
+    The web interface is accessible via port **8765**:
 
     - URL = `http://<your.IP>:8765`
     - User = `admin`
     - Password = not required
 
-    Passwords can be configured in the web interface
+    Passwords can be configured in the web interface.
 
 === "Access recordings (w/o web interface)"
 
-    Make sure you have one of [DietPi's file servers](../file_servers/) installed.  
+    To access you recordings remotely without the web interface, you might want to have one of [DietPi's file servers](../file_servers/) installed.  
     Directories used by MotionEye:
 
     - Media directory = `/mnt/dietpi_userdata/motioneye`
     - Accessed from file server = `/motioneye`
 
-=== "Upgrading"
+=== "Update to latest version"
 
-    MotionEye can be upgraded to latest version via
+    MotionEye can be updated to the latest version via
 
     ```sh
     sudo pip2 install -U motioneye
     ```
 
 ***
 
-Github page: <https://github.com/ccrisan/motioneye/wiki>  
-Tutorial: [MotionEye on DietPi on Raspberry Pi: keeping an eye on things](https://mansfield-devine.com/speculatrix/2018/12/motioneye-on-dietpi-on-raspberry-pi/)
+Github page: <https://github.com/ccrisan/motioneye>  
+Wiki: <https://github.com/ccrisan/motioneye/wiki>  
+Tutorial: [MotionEye on DietPi on Raspberry Pi: keeping an eye on things](https://mansfield-devine.com/speculatrix/2018/12/motioneye-on-dietpi-on-raspberry-pi/)  
+License: [GPLv3](https://github.com/ccrisan/motioneye/blob/dev/LICENSE)
+
+## mjpg-streamer
+
+Steam JPEG frames from various sources to various possible outputs. With the default setup it streams an attached camera image to a simple HTML page with the included webserver plugin.
+
+![mjpg-streamer HTML stream GIF](../assets/images/mjpg-streamer.gif){: width="500" height="400" loading="lazy"}
+
+=== "Access the HTML stream"
+
+    - Stream: `http://<your.IP>:8082/?action=stream`
+    - Snapshot: `http://<your.IP>:8082/?action=snapshot`
+    - Plugin documentation: <https://github.com/jacksonliam/mjpg-streamer/blob/master/mjpg-streamer-experimental/plugins/output_http/README.md>
```

Comment:
Should this documentation link not be moved below to the other docu/wiki links?

Reply:
Below is a link for all plugins. I though it's best here since this tab is about the HTML plugin explicitly. We might add tabs for other plugins, and then a quick link to each or their docs might be better than a long list below?

Reply:
Ok.